dist/src/main/resources/application-auth-oidc.yaml

```diff
@@ -0,0 +1,17 @@
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          oidcclient:
+            client-id:
+            client-secret:
+            authorization-grant-type: authorization_code
+            redirect-uri:
+            provider: oidcclient
+            scope:
```

Comment:
💭 Something for the future but we've had instance in the past where customers have wanted configurable scopes, I guess we can add this in later though if necessary

Reply:
I see. Doesn't the default way of property overriding for this case help? for example customer can override the scope property here with their new list of profile.

Reply:
Good point, it does, but i'm not sure if we want to make it a nicer experience down the line i.e. an env variable. Not critical for now at all but wanted to mention it